db/migrate/20191002180118_create_backups.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBackups < ActiveRecord::Migration[5.2]
+  def change
+    create_table :backups, id: :uuid do |t|
+      t.references :user, foreign_key: true, type: 'uuid'
```

Comment:
Doesn't look this field is being used or is needed?

Reply:
Deleted.